modules/local-kms/src/commonMain/sqldelight/com/sphereon/oid/fed/kms/local/models/1.sqm

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE Keys (
```

Comment:
@nklomp can you please advise on what is the ideal way to store this keys?

Reply:
Just store a JWK as private key. From that you can always get the public key

Reply:
And some shared secret to encrypt it would be really nice even though this is just a test KMS

Reply:
@nklomp Is it, like I should store JWK as string? Just store private key only?